.github/actions/python/action.yaml

```diff
@@ -8,27 +8,26 @@ inputs:
 runs:
   using: "composite"
   steps:
-
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
```

Comment:
I think we are supposed to use the `useblacksmith/setup-uv` action instead for some magical reason?

Reply:
all the `useblacksmith/setup-*` actions are going to be faster because they use their caches instead of GitHub's